src/main/groovy/org/labkey/gradle/task/DoThenSetup.groovy

```diff
@@ -94,7 +94,7 @@ class DoThenSetup extends DefaultTask
                             line = line.replace("@@ldapSyncConfig@@-->", "")
                             return line
                         }
-                        if (project.hasProperty("extraJdbcDataSource"))
+                        if (configProperties.hasProperty("extraJdbcDataSource"))
```

Comment:
The handling of this is at least different than the other properties mentioned here (e.g., `enableLdbapSync`).  I'm guessing either this check was incorrect before or the shape of the TeamCity properties has changed.  Can we add some comments to the getExtraJdbcProperties method to indicate what these properties are expected to look like?

Reply:
The check was incorrect before, and inconsistent between the embedded and non-embedded versions. I'm not sure who uses `enableLdapSync` and `enableJms` or when so I didn't want to touch them.
This change allows you to define a second datasource when deploying by specifying the jdbc properties in your `pg.properties` or `mssql.properties` files. Currrently, it assumes the properties will come if as Gradle properties (sufficient for TeamCity but kind of a pain when deploying locally).